source/includes/_tipser-react-elements.md

```diff
@@ -0,0 +1,263 @@
+# Tipser React Elements
+Tipser Elements is a React components library from Tipser enabling you to add shoppable content to your site with minimal effort.  You can add simple components like products with buy buttons, collections of products and articles, but also more complex grids or whole pages.
```

Comment:
> You can add simple components like products with buy buttons, collections of products and articles

Rather then writing about "articles" (which is a bit vague), I would rather mention a shop component in this sentence.

> but also more complex grids or whole pages
I guess it refers to our CMS module, right? But didn't we agree to not mention it in the docs for now? (there are still unanswered questions, like whether we want to have a shared or per-POS Contentful account or how do we convince our users to write parts of their CMS content in Contenful instead of their main CMS).

Reply:
Yes, I think that we shouldn't mention this CMS in the first place... it is not really prod-proof.